PWGHF/HFL/TableProducer/electronSelectionWithTpcEmcal.cxx

```diff
@@ -399,7 +537,93 @@ struct HfElectronSelectionWithTpcEmcal {
   {
     fillElectronTrack<true>(mcCollision, mcTracks, mcEmcClusters, matchedTracks, mcParticles);
   }
-  PROCESS_SWITCH(HfElectronSelectionWithTpcEmcal, processMcRec, "Process MC Reco mode", false);
+  PROCESS_SWITCH(HfElectronSelectionWithTpcEmcal, processMcRec, "Process MC Reco mode", true);
+
+  void processMcGen(aod::McParticles const& mcCollision, aod::McParticles const& mcParticles)
+  {
+
+    ///// electron identification
+    double ptMcGen = -999;
```

Comment:
if you use any variable only once, use the functional form directly. this will help to shorten the code

Reply:
okay